scripts/ci/deploy.sh

```diff
@@ -19,18 +19,37 @@
 set -e
```

Comment:
I renamed it to `deploy-aws-s3`. I'll rename it back to `deploy` once I manage to configure bintray snapshot builds.